highfidelity/hifi
# Notify sound inject

Test plan
- Open Tablet
- Press Snap button.
- Make sure there is shutter sound produced
- Switch audio output which is able to produce sound. 
- Repeat Snap
- Make sure shutter sound produced on new audio output
